Core/include/Acts/Seeding/SeedFinder.ipp

```diff
@@ -65,6 +65,10 @@ void SeedFinder<external_spacepoint_t, platform_t>::createSeedsForGroup(
 
   // Get the middle space point candidates
   const auto& middleSPs = grid.at(middleSPsIdx);
+  // Return if somehow there are no middle sp candidates
+  if (middleSPs.size() == 0) {
```

Comment:
Is this an error case? Or expected?

Reply:
can be expected if the user allows for it. We do not since we use the BinnedGroup class that skips empty bins. But it is still a possibility if the user does use it